binder/postBuild

```diff
@@ -16,18 +16,24 @@
 #
 set -euo pipefail
 
+export VERSION=1.22.4
```

Comment:
Could this be changed to `YARN_VERSION`?  I wasn't sure what version it was referring to until seeing the commands below.